BUILD.gn

```diff
@@ -303,10 +305,15 @@ source_set("manifests") {
   include_dirs = [ "//electron" ]
 
   deps = [
+    "buildflags",
     "//electron/shell/common/api:mojo",
     "//printing/buildflags",
     "//services/service_manager/public/cpp",
   ]
+
+  if (enable_builtin_spellchecker) {
+    deps += [ ":chrome_spellchecker" ]
```

Comment:
Would be good to just declare the mojom `//components/spellcheck/common:interfaces` target rather than complete target, minimal dependency. Also manifest based services will go away soon so this will only be short lived https://bugs.chromium.org/p/chromium/issues/detail?id=977637 :)